src/zino/tasks/reachabletask.py

```diff
@@ -50,8 +50,10 @@ async def _run_extra_job(self):
     def _update_reachability_event_as_reachable(self):
         event = self.state.events.get(self.device.name, None, ReachabilityEvent)
         if event and event.reachability != ReachabilityState.REACHABLE:
-            event.reachability = ReachabilityState.REACHABLE
+            event = self.state.events.checkout(event.id)
             event.add_log(f"{self.device.name} reachable")
+            event.reachability = ReachabilityState.REACHABLE
+            self.state.events.commit(event)
```

Comment:
This looks like an unrelated bugfix that should be in a PR of its own, or?

Reply:
I wouldnt call it unrelated since this is a necessary fix for the ac-down stuff to work properly for reachability events, but could be its own PR that this one depends on

Reply:
It should normally be - splitting things up always makes the PRs easier to review - but I'm going to let it slide this time, since I've already reviewed what I want to review.  Suggest @johannaengland  look at also :)